packages/office-ui-fabric-react/src/components/Panel/Panel.types.ts

```diff
@@ -77,6 +77,11 @@ export interface IPanelProps extends React.HTMLAttributes<PanelBase> {
    */
   headerText?: string;
 
+  /**
+   * The props for header text container.
+   */
+  headerTextProps?: IPanelHeaderTextProps;
```

Comment:
Is there a reason not to just make this `React.AllHTMLAttributes<HTMLDivElement>` now? If it needs to be expanded later, it could be broken out while still maintaining backwards interface compatibility. Less interfaces is more, I think.

Reply:
i don't have a strong reason. i will update this as you suggested